### PR TITLE
Change location to NULL island

### PR DESCRIPTION
**Pull request by RaresDumitrica**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -338,12 +338,10 @@ <h3>03 NOIEMBRIE</h3>
...
 				</div>
-				<div class="col-md-6">
+				<div class="col-md-6 section2-location" >
 					<h2>Locaţie</h2>
-					<p><i class="fa fa-map-marker" aria-hidden="true"></i>
- BUCUREŞTI</p>
- <p>Impact Hub Universitate</p>
- <p>Strada Tudor Arghezi, Nr. 8-10</p>
+					<a href="https://en.wikipedia.org/wiki/Null_Island"><p><i class="fa fa-map-marker" aria-hidden="true"></i> NULL ISLAND</p></a>
+					<p>Due to the global pandemic of Covid 19 this year's edition of <br/> Smarthack will take place online</p>
```
Comment (GabrielMajeri) [inline src/index.html:344]: Deși o variantă în engleză a site-ului ar fi șmecheră, deocamdată să păstrăm text-ul din pagină în română :sweat_smile:

Comment (RaresDumitrica) [reply src/index.html]: Am tradus inapoi in romana :))

**Comment by GabrielMajeri** [comment]
I've had time to look at this PR again, and I remembered how the initial idea was.

The plan was to add the Null Island location with a link up on top in the title. Here:
image

And down here we'd keep the text you've added for clarification, to make sure people don't actually try to go to Null Island :smile:
image

File: src/index.html (hunk 1 above)
Comment (GabrielMajeri) [reply src/index.html]: Arată bine textul, poți pune te rog și diacritice?

Comment (RaresDumitrica) [reply src/index.html]: Done, am adaugat locatia sus pe web si mobile si am pus o diacritica 👍

**Comment by GabrielMajeri** [comment]
I've added some `target="_blank"` to links to make them open in a new window. Everything else looks good!

Thanks for the PR :+1:

**Pull request merged**